tests/torchtune/modules/tokenizers/test_sentencepiece.py

```diff
@@ -7,18 +7,19 @@
 from pathlib import Path
 
 import pytest
-from torchtune.data._types import Message
-from torchtune.modules.tokenizers import SentencePieceTokenizer
+from torchtune.modules.tokenizers import SentencePieceBaseTokenizer
 
 ASSETS = Path(__file__).parent.parent.parent.parent / "assets"
 
 
-class TestSentencePieceTokenizer:
+class TestSentencePieceBaseTokenizer:
     @pytest.fixture
     def tokenizer(self):
         # m.model is a pretrained Sentencepiece model using the following command:
         # spm.SentencePieceTrainer.train('--input=<TRAIN_FILE> --model_prefix=m --vocab_size=2000')
-        return SentencePieceTokenizer(str(ASSETS / "m.model"))
+        sp_tokenizer = SentencePieceBaseTokenizer(str(ASSETS / "m.model"))
+        sp_tokenizer.encodes_whitespace = True
```

Comment:
Why do we need to set this explicitly?

Reply:
because the test model actually does not encode whitespaces and breaks the test. Alternatively, I can update the test values

Reply:
Sorry, what I mean is: why is this not already figured out by `SentencePieceBaseTokenizer`? Linking to where this should happen